postgresql/connection.go

```diff
@@ -0,0 +1,95 @@
+package postgresql
+
+import (
+	"database/sql"
+	"errors"
+	"fmt"
+	"os"
+
+	_ "github.com/GoogleCloudPlatform/cloudsql-proxy/proxy/dialers/postgres" // blank import
+	"github.com/joho/godotenv"
+
+	"github.com/benchttp/server/benchttp"
+)
+
+type computedStatsService struct {
+	db *sql.DB
+}
+
+func NewComputedStatsService(idleConn, maxConn int) (benchttp.ComputedStatsService, error) {
```

Comment:
It's generally preferable to return the actual struct rather than the interface it implements (although in our usage it's probably not going to be an issue)